src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/components/TradeQuoteContent.tsx

```diff
@@ -128,12 +128,14 @@ export const TradeQuoteContent = ({
             )}
           </Flex>
           <Skeleton isLoaded={!isLoading}>
-            <Amount.Fiat
-              color='text.subtle'
-              value={totalReceiveAmountFiatPrecision}
-              prefix='≈'
-              lineHeight={1}
-            />
+            {totalReceiveAmountFiatPrecision ? (
+              <Amount.Fiat
+                color='text.subtle'
+                value={totalReceiveAmountFiatPrecision}
+                prefix='≈'
+                lineHeight={1}
+              />
+            ) : null}
```

Comment:
question: this is to handle the case of no market data for custom assets correct?

Reply:
That's correct - in these cases we can't calculate `totalReceiveAmountFiatPrecision`, so we don't show it.